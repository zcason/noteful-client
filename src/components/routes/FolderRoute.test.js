import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import store from '../../dummy-store';
import FolderRoute from './FolderRoute';

describe(`FolderRoute component`, () => {
    const props = {
      store: store
    }
  
    it('renders FolderRoute by default', () => {
      const wrapper = shallow(<FolderRoute />)
      expect(toJson(wrapper)).toMatchSnapshot()
    })
  
    it('renders folder route from store when in props', () => {
      const component = shallow(<FolderRoute {...props} />)
        .find('.App')
      expect(toJson(component)).toMatchSnapshot()
    })
  })
  