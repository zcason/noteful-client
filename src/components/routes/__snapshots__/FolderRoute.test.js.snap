// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FolderRoute component renders FolderRoute by default 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`FolderRoute component renders folder route from store when in props 1`] = `null`;
